dev_guide/projected_volumes.adoc

```diff
@@ -0,0 +1,368 @@
+            - key: config
+              path: my-group/my-config
+              mode: 0777 <11>
+----
+
+<1> Add a `volumeMounts` section for each container that needs the secret.
+<2> Specify a path to an unused directory where the secret will appear.
+<3> Set `readOnly` to `true`.
+<4> Add a `volumes` block to list each projected volume source.
+<5> Specify any name for the volume.
+<6> Set the execute permission on the files.
+<7> Add a secret. Enter the name of the secret object. Each secret you want to use must be listed.
+<8> Specify the path to the secrets file under the `mountPath`. Here, the secrets file is in *_/projected-volume/my-group/my-config_*.
+<9> Add a Downward API source.
+<10> Add a ConfigMap source.
+<11> Set the mode for the specific projection
+====
+
+[NOTE]
+====
+If there are multiple containers in the pod, each container needs a `volumeMounts` section, but only one `volumes` section is needed.
+====
+
+
+.Pod with multiple secrets with a non-default permission mode set
+====
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      defaultMode: 0755
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - secret:
+          name: mysecret2
+          items:
+            - key: password
+              path: my-group/my-password
+              mode: 511
+----
+
+====
+
+[NOTE]
+====
+The `defaultMode` can only be specified at the projected level and not for each
+volume source. However, as illustrated above, you can explicitly set the `mode`
+for each individual projection.
+====
+
+## Pathing Considerations
+
+When creating projected volumes, consider the following situations related to the volume file paths.
+
+### Collisions between keys when configured paths are identical
+
+If you configure any keys with the same path, the pod spec will not be accepted as valid.
+In the following example, the specified path for `mysecret` and `myconfigmap` are the same:
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/data
+      - configMap:
+          name: myconfigmap
+          items:
+            - key: config
+              path: my-group/data
+----
+
+### Collisions between keys without configured paths
+
+The only run-time validation can occur is when all the paths are known at pod creation, similar to the above scenario.
```

Comment:
Something is missing from the above. I think a "that" after "validation"?

"The only run-time validation that can occur is when all the paths are known at pod creation..."